.rubocop.yml

```diff
@@ -17,18 +17,19 @@ Lint/ImplicitStringConcatenation:
     - 'lib/generators/sufia/**/*'
 
 Metrics/LineLength:
-  Max: 400
+  Enabled: false
 
 Metrics/AbcSize:
-  Max: 30
-  Exclude:
-    - 'app/controllers/concerns/sufia/admin/depositor_stats.rb'
-    - 'app/controllers/concerns/sufia/my_controller_behavior.rb'
-    - 'app/services/sufia/user_stat_importer.rb'
-    - 'lib/sufia/arkivo/metadata_munger.rb'
+  Enabled: false
 
 Metrics/MethodLength:
-  Max: 15
+  Enabled: false
```

Comment:
Sure if you want to discuss further.  I've chosen to disable using https://github.com/psu-stewardship/scholarsphere/blob/develop/.rubocop.yml as a template for the decisions I made.